### ISSUE TITLE
onnx runtime install error

### ISSUE THREAD

**Issue by fideliochan** [Snd-R/Komelia#3]
I have iot 21h2 w10, have cuda 12 and 12.5 i get this error no matter what i try:
```
Failed to load ONNX Runtime:C:\Users \REDACTED\AppData\Local\Temp\komelia_libs\onnxruntime_providers_cuda.dIl: Can't find dependentlibraries
```

**Comment by Snd-R** [comment]
Cuda installation is know to have problems, it won't work in latest release. It also requires cudnn 8 install

The library loading issue was already fixed in the main branch. but there's still a few problems with upscaling
- fp16 models are not supported in latest release (already fixed in main branch)
- no auto tiling so for big images you might run out of vram (work in progress)

You can use DirectMl in the meantime, performance should still be decent

**Comment by Snd-R** [comment]
Should be fixed in 0.6.0
https://github.com/Snd-R/Komelia/releases/tag/0.6.0
